.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-18.04
```

Comment:
- [ Optional] Please you have created the likes reference key before creating the table itself this cause a problem.
![image](https://user-images.githubusercontent.com/74744787/186273312-e14f7cb2-6c50-4063-8613-180554c82482.png)